src/language/utils.ts

```diff
@@ -117,6 +117,24 @@ export function getBindingElementVariableDeclaration(node: ts.BindingElement): t
     return <ts.VariableDeclaration> currentParent;
 }
 
+/** Shim of Array.find */
```

Comment:
We don't need this shim -- `Array.prototype.find` is supported on Node v4+, and we only support 4.2+ (see [circle.yml](https://github.com/palantir/tslint/blob/a7cbdef1008e4193653275493c746ae1861d1f76/circle.yml#L7)).